promadapter/conversion.go

```diff
@@ -5,22 +5,38 @@ import (
 
 	"time"
 
+	"github.com/appoptics/appoptics-api-go"
 	"github.com/prometheus/common/model"
 	promremote "github.com/prometheus/prometheus/storage/remote"
```

Comment:
warning: exported type PrometheusAdapter should have comment or be unexported (golint)